frontend/src/components/landing/Landing.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState, ReactElement } from "react";
+import cx from "classnames";
+import styles from "./Landing.module.css";
+
+import Button from "../atoms/button/Button";
+
+import LandingPageTemplate from "../../pages/landingPageTemplate/LandingPageTemplate";
+import introScreen from "../../assets/images/intro-screen.png";
+import registrationScreen from "../../assets/images/registration-screen.png";
+import upgradePage from "../../assets/images/upgrade-page.png";
+import keyboardScreen from "../../assets/images/keyboard-screen.png";
+
+import getFurtherUrl from "../../services/getFurtherUrl";
+import getPythonSDKDocsUrl from "../../services/getPythonSDKDocsUrl";
+import openKnowledgeBase from "../../services/openKnowledgeBase";
+import openFurther from "../../services/openFurther";
+import openPythonSDKDocs from "../../services/openPythonSDKDocs";
+
+export type UrlData = {
+  defaultUrl: string;
+  urlService?: () => Promise<{ [s: string]: string }>;
+  onWebRenderer: () => Promise<void>;
+};
+
+export type LandingPageElement = {
+  title: string;
+  message: string;
+  prompt: ReactElement;
+  image: string;
+  urlInfo: UrlData;
+  buttonLabel?: string;
+};
+
+const pages = [
+  {
+    title: "Learn by making on Further",
+    id: "further",
+    detail: (
+      <LandingPageTemplate
+        key="further"
+        title="Learn by making on Further"
+        urlInfo={{
+          onWebRenderer: openFurther,
+          defaultUrl: "http://further.pi-top.com",
+          urlService: getFurtherUrl,
+        }}
+        message="A community of pi-top owners who like hands on learning.  It's time to get you started and show you a bit about how Further works. Grab your pi-top and hit next to continue."
+        prompt={<>Learn by making on <span className="green">Further =)</span></>}
+        image={introScreen}
+      />
+    ),
+  },
+  {
+    title: "Python SDK",
+    id: "sdk",
+    detail: (
+      <LandingPageTemplate
+        key="sdk"
+        title="Python SDK"
+        urlInfo={{
+          defaultUrl: "http://docs.pi-top.com",
+          urlService: getPythonSDKDocsUrl,
+          onWebRenderer: openPythonSDKDocs,
+        }}
+        message={
+          'The Software Development Kit (SDK) provides an easy-to-use framework to interact with your pi-top using python. It also contains CLI utilities to manage your pi-top using the terminal.\nPress the "Let\'s Go" button to open its documentation and start making!'
+        }
```

Comment:
As a fragment this can include the `br` elements inline:

```suggestion
        message={<>
          The Software Development Kit (SDK) provides an easy-to-use framework to interact with your pi-top using python. It also contains CLI utilities to manage your pi-top using the terminal.<br />Press the "Let's Go" button to open its documentation and start making!
        </>}
```